content/applications/productivity/studio/models_modules_apps.rst

```diff
@@ -0,0 +1,216 @@
+=========================
+Models, modules, and apps
+=========================
+
+Models, also known as *Objects*, determine the logical structure of a database and how data is
```

Comment:
Are models labeled as "Objects" anywhere in Odoo? Because they should not. A model is used to define the structure of data; technically, it's a class. A record is the instanced data of a model; technically, it's an object.

Reply:
When you use the Studio app creator 'wizard' (Choose your first _object_ name):

<img width="237" alt="chrome_2xnTboKcdH" src="https://user-images.githubusercontent.com/70884563/181198400-15018c7e-7c0b-46cd-a0eb-b0a6783e683c.png">

Or when you activate debug mode and hover on a field:

<img width="380" alt="chrome_1nTGWXRruJ" src="https://user-images.githubusercontent.com/70884563/181198612-b6964bb1-763a-40fd-aa05-cf1c72766500.png">

Now I can definetly remove the reference to objects to avoid any confusion.

Reply:
Oh gawd. Then it's up to you I guess, but I'm disappointed that even the debug mode uses the term "object"^^

Reply:
..._also sometimes wrongly referred to as *Objects*_...